features/admin.extensions.v1/configs/components/my-account-overview.tsx

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { IdentifiableComponentInterface } from "@wso2is/core/models";
+import {
+    CopyInputField,
+    DocumentationLink,
+    EmphasizedSegment,
+    Heading,
+    Link,
+    Popup,
+    Text
+} from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import { Grid } from "semantic-ui-react";
+import { AppConstants, AppState, history } from "../../../admin.core.v1";
+
+export const MyAccountOverview: FunctionComponent<IdentifiableComponentInterface> = (
+    props: IdentifiableComponentInterface
+): ReactElement => {
+    const { ["data-componentid"]: componentId } = props;
+    const consumerAccountURL: string = useSelector((state: AppState) =>
+        state?.config?.deployment?.accountApp?.tenantQualifiedPath);
+    const { t } = useTranslation();
+
+    return (
+        <EmphasizedSegment padded="very">
+            <div data-componentid={ componentId } className="form-container">
+                <Heading data-componentid={ componentId } bold as="h4">
+                    { t("applications:myaccount.overview.heading") }
+                </Heading>
+                <Text>
+                    { t("applications:myaccount.overview.contentIntro") }
+                    <DocumentationLink
+                        link={ "develop.applications.myaccount.overview.learnMore" }
+                        isLinkRef = { true }
```

Comment:
We do not need to explicitly set to true.

```suggestion
                        isLinkRef
```

Reply:
When this is not set the browser will be redirected even if the value is undefined. in this case it will navigate to `console/applications/develop.applications.myaccount.overview.learnMore`. When set the link is not clickable when the url is undefined. so better to set this

Reply:
What i meant is leaving `isLinkRef` as it is, is equivalent to `isLinkRef={ true }`. Therefore, we can safely remove `={ true }` part here. 🙂